app/models/comment.rb

```diff
@@ -1,4 +1,6 @@
 class Comment < ApplicationRecord
   belongs_to :post
   belongs_to :user
+
+  delegate :name, to: :user
```

Comment:
giả sử sau này comment cũng có 1 attribute là `#name` thì khi đó call `Comment.find(xxx).name` nó sẽ ra cái gì em ?

Reply:
làm sao đặt được tên trường như kia nhỉ anh nhỉ :|

Reply:
ý em là có # ở trước

Reply:
còn nếu là trường name bình thường, thì nếu viết Comment.find(xxx).name nó sẽ lấy thẳng name của user ạ

Reply:
@leduccuong1612 làm gì có trường nào # ở đằng trc dc e
ý hưng là giả sử comment có 1 trường là name. vs code delegate :name, to: :user này khi comment.name thì nó sẽ lấy name nào, name của comment hay name của user

Reply:
vâng thì nó lấy name của user ạ

Reply:
e nên dùng prefix nữa nhé. để biết đó là user_name. chứ comment.name thì biết name nào

Reply:
ồ, vâng ạ